docs/functions/functions.json

```diff
@@ -1424,6 +1572,69 @@
 				true
 			]
 		},
+		{
+			"name": "filename",
+			"id": "filename",
+			"unary": false,
+			"category": "other",
+			"role": "function",
+			"argument": [
+				{
+					"name": "URL",
+					"optional": false,
+					"type": "Text"
+				}
+			],
+			"description": "Gets all texts after the rightmost slash of an URL. For example, `filename(\"http://mavo.io/docs/functions.html\")` will return `functions.html`.\nIf the parameter is not an URL, `filename()` will return the value of the parameter as-is.",
+			"returnType": "Text",
+			"returnDescription": "Text after the rightmost slash of an URL. Returns the value of the parameter as-is if not an URL."
+		},
+		{
+			"name": "json",
+			"id": "json",
+			"unary": false,
+			"category": "other",
+			"role": "function",
+			"description": "Converts data into JSON string and avoids circular reference.",
+			"argument": [
+				{
+					"name": "data",
+					"optional": false,
+					"type": "Anything",
+					"description": "Data to be converted to JSON string."
+				}
+			],
+			"returnType": "Text",
+			"returnDescription": "The JSON string."
+		},
+		{
+			"name": "log",
+			"id": "log",
+			"unary": false,
+			"category": "other",
+			"role": "function",
+			"description": "Logs the result of all parameters to the browser console and returns the result of the first parameter of this function.\nFor example, `log(1 + 1, floor(3.5), true && false)` will return `2`, the result of the first parameter of `log()`. Additionally, when you open the browser console, `2 3 false` will be displayed.",
```

Comment:
Maybe mention that this is primarily intended for advanced users and developers and link to the Mavo inspector as something that may be more useful?